### PR TITLE
LPD-25814 | master

### PR DESCRIPTION
**Pull request by marcosapmf**
Jira ticket: https://liferay.atlassian.net/browse/LPD-25814

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:sf

File: modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts [hunk 1]
```diff
@@ -76,6 +77,24 @@ export async function syncAllContacts(page) {
 	await page.getByRole('button', {exact: true, name: 'Next'}).click();
 }
 
+export async function syncAnalyticsCloud(page) {
+	await createDataSource(page);
+
+	await goToAnalyticsCloudInstanceSettings(page);
+
+	await acceptsCookiesBanner(page);
```
Comment (ViniLopes20) [inline modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts:85]: Hey @marcosapmf, Will the cookies banner appear in your test?

File: modules/test/playwright/tests/osb-faro-web/page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import {syncAnalyticsCloud} from '../analytics-settings-web/utils/analyticsSettings';
+import {faroConfig} from './faro.config';
+
+export const test = mergeTests(loginAnalyticsCloudTest(), loginTest());
+
+test('renders workspace list', async ({page}) => {
```
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/page.spec.ts:16]: Hey @marcosapmf, can you change the test name to be more related to the test proposal?

File: modules/test/playwright/tests/osb-faro-web/page.spec.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+		timeout: 100 * 1000,
+	});
+
+	await expect(page.getByText('1', {exact: true}).first()).toBeVisible({
+		timeout: 100 * 1000,
+	});
+
+	await expect(page.getByText('1', {exact: true}).nth(2)).toBeVisible({
+		timeout: 100 * 1000,
+	});
```
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/page.spec.ts:72]: Hey @marcosapmf, you need to generate an exit page to an external página of the DXP portal

**Review by ViniLopes20 — CHANGES_REQUESTED** [review]
Thanks @marcosapmf, Very good job with your workaround, hahaha. Could you check my comments?

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:analytics-cloud

File: modules/test/playwright/tests/osb-faro-web/page.spec.ts [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+	await page
+		.getByRole('link', {
+			name: 'FARO-DEV-liferay Liferay Demo Enterprise Plan',
+		})
+		.click();
+
+	await page.locator('.channels-menu.button-root').click();
+
+	await page.locator('.dropdown-item.sites-dropdown-item').last().click();
```
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/page.spec.ts:90]: Hey @marcosapmf, I think you could get the name of the property that you will use in your tests before executing the `await syncSite(page);` in the `syncAnalyticsCloud` function. This way, you will know the name of the property that you will need to search for and select in the dropdown.

**Review by ViniLopes20 — CHANGES_REQUESTED** [review]
Hey @marcosapmf, Could you check my comment?

**Comment by marcosapmf** [comment]
ci:test:analytics-cloud

**Comment by marcosapmf** [comment]
ci:test:sf

File: modules/test/playwright/tests/analytics-settings-web/analyticsSettings.spec.ts [hunk 5]
```diff
@@ -7,32 +7,10 @@ import {mergeTests} from '@playwright/test';
...
-	await disconnectFromAnalyticsCloud(page);
-
-	await connectToAnalyticsCloud(page);
-
-	await syncSite(page);
-
-	await syncAllContacts(page);
-
-	await page.getByRole('button', {name: 'Finish'}).click();
+	syncAnalyticsCloud(page, 'My Property');
```
Comment (ViniLopes20) [inline modules/test/playwright/tests/analytics-settings-web/analyticsSettings.spec.ts:15]:
```suggestion
        const channelName = 'My Property - ' + getRandomString();
	syncAnalyticsCloud(page, channelName);
```

**Review by ViniLopes20 — CHANGES_REQUESTED** [review]

**Comment by marcosapmf** [comment]
ci:forward

**Review by ViniLopes20 — APPROVED** [review]
Good Job, @marcosapmf 🎉

**Comment by marcosapmf** [comment]
Manually forwarded to https://github.com/brianchandotcom/liferay-portal/pull/150486

**Pull request closed without merging**